plugin/pom.xml

```diff
@@ -68,6 +68,30 @@
       <version>1.10.6</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-databind</artifactId>
+          <version>2.9.8</version>
+          <scope>compile</scope>
+      </dependency>
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>credentials</artifactId>
+          <version>2.2.0</version>
+          <scope>compile</scope>
+      </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
```

Comment:
not sure why you had to add this dep and the next one?

Reply:
Yeah I will get rid of it